File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

for i in range(n):
    for j in range(n):
        if i + j >= n - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()

# ANOTHER OPTIONS

# for i in range(n):
#   for j in range(n):
#       print("#" if i + j >= n - 1 else " ", end="")
#   print()

# for i in range (n):
#    print((n - 1 - i) * " ", end="")
#    print((i + 1) * "#")

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // asks for input between 1-9
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    // iterates through height inputted
    for (int i = 0; i < height; i++)
    {
        // print spaces
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        // print hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // new line
        printf("\n");
    }
}